src/Tizen.Content.Download/Tizen.Content.Download/Request.cs

```diff
@@ -843,6 +844,213 @@ private void UnregisterProgressChangedEvent()
                 DownloadErrorFactory.ThrowException(ret, "Unsetting ProgressChanged callback failed");
             }
         }
+
+        /// <summary>
+        /// Sets the 'enabled' state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <param name="enable">The enable value</param>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <remarks>
+        /// Enable/disable for _downloadId
```

Comment:
'_downloadId' is a private value. It's not appropriate to show in the API level.

Reply:
> '_downloadId' is a private value. It's not appropriate to show in the API level.

Updated. Thanks.